trino/client.py

```diff
@@ -1185,6 +1186,14 @@ def map(self, values: Any) -> Optional[Dict[Any, Optional[Any]]]:
         }
 
 
+class UuidValueMapper(ValueMapper[uuid.UUID]):
+    def map(self, values: Any) -> Optional[uuid.UUID]:
+        try:
+            return uuid.UUID(values)
+        except:
+            return None
```

Comment:
Why the exception handling,  I don't think we would like to silently discard values.

```suggestion
        return uuid.UUID(values)
```

Reply:
The test currently expects that `CAST(NULL AS UUID)` returns `None` in Python. Should we expect a different behavior or a specific case should be applied when `values` is `None`?

Reply:
I get it but we should not let `None` go into UUID, similar like all the other mappers.